documentation.html

```diff
@@ -525,6 +525,8 @@ <h4 class="display-inlineBlock">Adding dark-mode as an attribute</h4><a  href="#
   <span class="scroll-top" data-scroll="up" style="display: none;">
     <i class="fa fa-chevron-circle-up" aria-hidden="true"></i>
   </span>
+<<<<<<< HEAD
```

Comment:
Please remove all new lines containing "<<<<<" or ">>>>" or "====" or anything similar as it is the result of issues in merging